infra/examples-dev/aws-msft-365/main.tf

```diff
@@ -70,8 +70,12 @@ module "psoxy-aws-msft-365" {
   certificate_subject            = var.certificate_subject
   pseudonymize_app_ids           = var.pseudonymize_app_ids
   general_environment_variables  = var.general_environment_variables
+#  aws_ssm_key_id                 = aws_kms_key.key.key_id
 }
 
+#resource "aws_kms_key" "key"  {
+#}
```

Comment:
I'd copy the full spec just in case

```suggestion
## TODO: requires targeted apply to create key first, bc value of key_id determines map content
## in example
#resource "aws_kms_key" "key" {
#  description             = "KMS key for Psoxy"
#  enable_key_rotation     = true
#  is_enabled              = true
#}
```